metabrainz/templates/index/code-of-conduct.html

```diff
@@ -3,6 +3,13 @@
 {% block title %}{{ _('Code of Conduct') }} - MetaBrainz Foundation{% endblock %}
 
 {% block content %}
+  {###
+    When updating this file, please also update the other CODE_OF_CONDUCT.md files in other repositories:
+      * The most important is for the template repository as it is automatically used in other repositories not having such file:
```

Comment:
"The most important is the template repository"? (not sure if the for was intended)

Reply:
Thanks, removed in commit https://github.com/metabrainz/metabrainz.org/pull/379/commits/a3f7c6ca0ccd7865a88cdfc874ef25ead3bb4900.